packages/perseus/src/__tests__/renderer-api.test.tsx

```diff
@@ -165,11 +164,7 @@ describe("Perseus API", function () {
                 // version
                 expect(ClassNames.FOCUSED).toBe("perseus-focused");
 
-                const renderer = renderQuestionArea(inputNumber1Item, {
-                    interceptInputFocus: function (widgetId) {
```

Comment:
I checked and this API option hasn't existed since we returned to this repo over a year ago. The test still passes with this removed, so 🤷‍♂️ .